app/controllers/ctc/portal/portal_controller.rb

```diff
@@ -9,10 +9,12 @@ def home
       @status = latest_transition.present? ? latest_transition.to_state : EfileSubmissionStateMachine.initial_state
       @exposed_error = latest_transition.present? ? latest_transition.client_facing_errors.first : nil
       @current_step = nil
+      @pdf1040 = current_client.documents.find_by(tax_return: submission.tax_return, document_type: DocumentTypes::Form1040.key)
```

Comment:
Should we be concerned about the possibility that there's more than one, for example, if the Hub user edits their data, then resubmits?

Not a blocker.

Reply:
we use the same logic we use for creating new versions of the 136146, which is that we just say "if there is already a 1040 for this tax return, regenerate and reattach the file to the same document object". So there is only ever one!